### PR TITLE
FF101 TransformStream API updates

### PR DESCRIPTION
**Pull request by hamishwillee**
FF101 adds support to FF Nightly for [TransformStreamDefaultController](https://developer.mozilla.org/en-US/docs/Web/API/TransformStreamDefaultController),[ReadableStream.pipeThrough](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/pipeThrough) via a pref.

This updates the associated docs
- Add to experimental features
- General consistency and front matter fixes to above docs.

The docs appear to match the spec and have reasonable examples etc.

Other docs work for this can be tracked in #15469

### CODE REVIEW

File: files/en-us/web/api/transformstreamdefaultcontroller/enqueue/index.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ enqueue(chunk)
 
 None ({{jsxref("undefined")}}).
 
+### Exceptions
+
+- `TypeError`
```
Comment (hamishwillee) [inline files/en-us/web/api/transformstreamdefaultcontroller/enqueue/index.md:35]: Note that other errors can be thrown here. However working out what they are from the spec would be a nightmare. This is the most likely to be encountered IMO.

Comment (teoli2003) [inline files/en-us/web/api/transformstreamdefaultcontroller/enqueue/index.md:35]:
```suggestion
- {{jsxref("TypeError")}}
```

**Comment by hamishwillee** [comment]
@wbamberg @teoli2003 Anything I need to do to move this along?

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -937,11 +937,53 @@ With this feature enabled, Firefox supports [JPEG XL](https://jpeg.org/jpegxl/)
...
-      <td colspan="2">image.jxl.enabled</td>
+      <td colspan="2"><code>image.jxl.enabled</code></td>
     </tr>
   </tbody>
 </table>
 
+
+#### Streams API: TransformStreams
+
+Firefox support for the [Streams API](/en-US/docs/Web/API/Streams_API) transform streams, including classes and methods: [`TransformStream`](/en-US/docs/Web/API/TransformStream), [`TransformStreamDefaultController`](/en-US/docs/Web/API/TransformStreamDefaultController), and [`ReadableStream.pipeThrough()`](/en-US/docs/Web/API/ReadableStream/pipeThrough).
```
Comment (Rumyra) [inline files/en-us/mozilla/firefox/experimental_features/index.md:948]: I'm not sure if the first part of this sentence reads quite right...

**Review by Rumyra — CHANGES_REQUESTED** [review]

**Comment by Rumyra** [comment]
Reviewed @hamishwillee sorry for the delay

File: files/en-us/mozilla/firefox/experimental_features/index.md (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/experimental_features/index.md:948]:
```suggestion
Support for [transform streams](/en-US/docs/Web/API/Streams_API#transform_streams), including the classes [`TransformStream`](/en-US/docs/Web/API/TransformStream) and [`TransformStreamDefaultController`](/en-US/docs/Web/API/TransformStreamDefaultController), and the method [`ReadableStream.pipeThrough()`](/en-US/docs/Web/API/ReadableStream/pipeThrough).
```

Comment (hamishwillee) [reply files/en-us/mozilla/firefox/experimental_features/index.md:948]: Thanks @Rumyra , I think fixed by the above ^^^

**Review by Rumyra — APPROVED** [review]

**Pull request merged**